lib/preload_page_view.dart

```diff
@@ -551,15 +589,18 @@ class _PreloadPageViewState extends State<PreloadPageView> {
         physics: physics,
         viewportBuilder: (BuildContext context, ViewportOffset position) {
           return Viewport(
-            cacheExtent: _preloadPagesCount <= 1
-                ? 1
+            cacheExtent: _preloadPagesCount < 0
```

Comment:
There are a issue here, it should be:
```
_preloadPagesCount < 1
? 0
: (_preloadPagesCount ==1 
    ? 1
    : ((widget.scrollDirection == Axis.horizontal
        ? MediaQuery.of(context).size.width * _preloadPagesCount - 1
        : MediaQuery.of(context).size.height * _preloadPagesCount -1),))
```
BUT, due to it became soooo complicate, perhaps we could create some variable and use some if statement to do that.

Reply:
I think a function like `_calculateCacheExtent()` that takes all of these variables into account and returns proper cacheExtent value would make the code cleaner.